Week-5/5.2/index.js

```diff
@@ -0,0 +1,20 @@
+function isVowel(char) {
+  return ["a", "e", "i", "o", "u"].includes(char);
+}
+
+function vowelCount(str) {
+  const vowelMap = new Map();
```

Comment:
Instead of empty map, can create an initial map with a,e,i,o,u and initialise value to 0.